connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/StreamingForeachBatchHelper.scala

```diff
@@ -44,16 +44,16 @@ object StreamingForeachBatchHelper extends Logging {
       sessionHolder: SessionHolder): ForeachBatchFnType = { (df: DataFrame, batchId: Long) =>
     {
       val dfId = UUID.randomUUID().toString
-      log.info(s"Caching DataFrame with id $dfId") // TODO: Add query id to the log.
+      logInfo(s"Caching DataFrame with id $dfId") // TODO: Add query id to the log.
 
-      // TODO: Sanity check there is no other active DataFrame for this query. The query id
-      //       needs to be saved in the cache for this check.
+      // TODO(SPARK-44462): Sanity check there is no other active DataFrame for this query.
```

Comment:
Added spark tickets to each TODO to make sure we don't forget. It could be that the ticket and the comment content doesn't match. cc @rangadi